Eradication/VectorPopulationIndividual.cpp

```diff
@@ -599,8 +600,10 @@ namespace Kernel
         else if( m_UnmatedMaleTotal > 0 )
         {
             auto it = SelectMaleMatingCohort();
+            VectorCohortMale* p_male_cohort = static_cast<VectorCohortMale*>(static_cast<VectorCohortAbstract*>(*it));
+            release_assert(p_male_cohort->GetUnmatedCount() > 0);
+            UpdateMaleMatingCDF(it); // updating this here so when it comes to the new microsporidia cohort, the numbers are all updated already (like it VectorPopulation)
```

Comment:
make put comment on its own line so line isn't so long

"like it VectorPopulation" to "like in VectorPopulation" - you're correcting my typos :)